src/components/auth/torus/AuthTorus.web.js

```diff
@@ -190,37 +280,35 @@ const AuthTorus = ({ screenProps, navigation, styles, store }) => {
         const { source } = await ready(replacing)
 
         log.debug('showing checkmark dialog')
+
         // showLoadingDialog(true)
         // await delay(30000000)
 
         // await new Promise(res => showLoadingDialog(true, res))
         // showLoadingDialog(true)
         // log.debug('hiding checkmark dialog')
         hideDialog()
-
-        //user exists reload with dashboard route
+        if (isSignUp) {
+          if (exists) {
+            return showEmailUsedDialog(provider, source)
+          }
+          fireEvent(SIGNUP_STARTED, { source, provider })
+          return navigate('Signup', {
+            regMethod: REGISTRATION_METHOD_TORUS,
+            torusUser,
+            torusProvider: provider,
+          })
+        }
         if (exists) {
+          successDialog()
+          await delay(2000)
```

Comment:
what is this delay?

Reply:
It's to show the success dialog before redirecting to the dashboard.

Reply:
where is it part of the requirements?
remove it and make sure it is same behaviour just like it is on master